src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1255,6 +1265,7 @@ data CompositeKeyError =
   | MakeSetError
   | MakeWhereError
   | MakeHavingError
+  | BetweenError
```

Comment:
I think we can remove this when `CompositeKey` is supported.

Reply:
Shouldn't we throw this error for MySQL since it can't be used anyways as you just mentioned?

Reply:
I'd rather make it work for MySQL if at all possible, even if we have to do the expanded form.